src/FilterPanel.jsx

```diff
@@ -31,13 +31,52 @@ export class FilterPanel extends React.Component {
             <input type="text" placeholder="Enter company name" />
         </CollapsibleFilter>
         <hr />
+        <Aggregation title="Product / sub-product"
+                     desc="The type of product and sub-product the consumer identified in the complaint"
+                     options={this.props.aggs.product}
+        />
+        <hr />
+        <Aggregation title="Issue / sub-issue"
+                     desc="The type of issue and sub-issue the consumer identified in the complaint"
+                     options={this.props.aggs.issue}
+        />
+        <hr />
+        <CollapsibleFilter title="State"
+                           desc="The state of the mailing address provided by the consumer">
+            <input type="text" placeholder="Enter state name or abbreviation" />
+        </CollapsibleFilter>
+        <hr />
+        <CollapsibleFilter title="Zip Code"
+                           desc="The mailing ZIP code provided by the consumer">
+            <input type="text" placeholder="Enter first three digits of zip code" />
+        </CollapsibleFilter>
+        <hr />
         <Aggregation title="Did company provide a timely response?"
-                         options={this.props.aggs.timely_response}
+                     options={this.props.aggs.timely}
```

Comment:
My current plan with #15 is to use a `fieldName` prop that passes the name of the aggregation field as we need that to update params: `fieldName='timely'`

Any issues adding that to `Aggregatin` moving forward or is there a better way to derive it from the component with the current props structure that I should be using?